web/pdf_viewer.css

```diff
@@ -39,11 +44,49 @@
 
 .pdfViewer.singlePageView {
   display: inline-block;
+  margin: 0;
```

Comment:
This CSS is not used by the default viewer, but by `examples/components/pageviewer.html`.
Are the examples also part of your test plan? We should indeed not regress on the single page view case.